lib/components/home/artwork_rail.js

```diff
@@ -50,6 +58,7 @@ export default Relay.createContainer(ArtworkRail, {
         results @include(if: $fetchContent) {
           __id
           title
```

Comment:
Well I do need the rail title. It could be moved into `generic_grid`, but I think it makes sense to leave it in the rail since the title styling is specific to Home.